src/main/java/com/epam/tc/hw7/entities/Data.java

```diff
@@ -4,13 +4,13 @@
 
 public class Data {
```

Comment:
Data is a very generic name, should be specified for real content

Reply:
changed